mlflow/utils/workspace_utils.py

```diff
@@ -1,5 +1,94 @@
-"""Workspace-related utility constants."""
+"""Workspace-related utility constants and helpers."""
```

Comment:
I think we can get rid of the module-level docstring. The name says it all. 

The rest of the docstrings in this file could benefit from standard Google style docstrings (args, returns), though.